src/index.js

```diff
@@ -2,48 +2,24 @@
 import _ from 'lodash';
 /* eslint-enable */
 import './style.css';
+import { addList, displayList } from './modules/app.js';
+import localGet from './modules/localstorage.js';
 
-const toDoList = [
-  {
-    description: 'hello world',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'wash plate',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'cook beans',
-    completed: true,
-    index: 3,
-  },
-];
+const form = document.querySelector('.list-con');
+const todoTask = document.getElementById('input');
 
-toDoList.forEach((e) => {
-  const { description, completed, index } = e;
-  const ulLists = document.querySelector('.ul-items');
+window.addEventListener('load', () => {
+  displayList();
+});
 
-  const list = document.createElement('li');
-  list.setAttribute('class', 'list');
-  list.setAttribute('id', index);
-  ulLists.appendChild(list);
+// addBtn.addEventListener('click', (event) => {
+//   event.preventDefault();
+//   addList(todoTask.value, false, localGet.length + 1);
+//   displayList();
+// })
```

Comment:
It's highly recommended to remove the commented code from your project, please make sure to keep your repository as clean as possible.